server/sonar-server/src/main/java/org/sonar/server/computation/ws/SubmitReportWsAction.java

```diff
@@ -72,16 +68,21 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
+    UserSession.get().checkGlobalPermission(GlobalPermissions.SCAN_EXECUTION);
```

Comment:
where do we want to manage user permissions most of the time ? ws level ? I've heard different opinions.
I'm fine with WS level, I'm just concerned with consistency